content/library/api/connections/connections-sql.md

```diff
@@ -0,0 +1,16 @@
+---
+title: st.connections.SQLConnection
```

Comment:
Is this what controls this string? If so - change to just `SQLConnection` (and same for SnowparkConnection title)
![Screenshot 2023-04-24 at 5 16 17 PM](https://user-images.githubusercontent.com/116604821/234143106-db169ae1-a31c-4e28-83b3-8a303a54a4be.png)

Reply:
We cannot and shouldn't change it here. If we do that, we have to do it for other commands in our docs. `st.line_chart` -> `line_chart`, `st.button` -> `button`, etc.

Reply:
> Is this what controls this string?

[python/generate.py](https://github.com/streamlit/docs/pull/622/files#diff-7a46e933a239e9d07959869c6d02169104f52aa74a306d4ec86333eeb8a624bf) and [components/blocks/autofunction.js](https://github.com/streamlit/docs/pull/622/files#diff-b34f10e40cfe9739517ed77ead96fbad855131783636ff85c23e45df28904796)

Reply:
Ah OK makes sense!